python/spider-py/src/spider_py/client/task_context.py

```diff
@@ -0,0 +1,25 @@
+"""Spider client task context module."""
+
+from spider_py import core, storage
+from spider_py.client.data import Data
+
+
+class TaskContext:
+    """Spider task context provides access to the task id and data creation for tasks."""
+
+    def __init__(self, task_id: core.TaskId, storage: storage.Storage) -> None:
+        """Initializes the task context."""
+        self._task_id = task_id
+        self._storage = storage
+
+    @property
+    def task_id(self) -> core.TaskId:
+        """:return: The task id."""
+        return self._task_id
+
+    def create_data(self, data: Data) -> None:
+        """
+        Creates a data in the storage.
```

Comment:
```suggestion
        Creates a task-ID referenced data object in the storage.
```
"a data" is a grammar mistake.